common/nymcoconut/src/constants.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 Nym Technologies SA
```

Comment:
1. This is old license
2. This file doesn't seem to have anything useful in it? : )